src/apps/viewer/extensions/cellpack/curve.ts

```diff
@@ -191,9 +191,9 @@ function GetMiniFrame(points: Vec3[], normals: Vec3[]) {
 
 const rpTmpVec1 = Vec3()
 
-export function getMatFromResamplePoints(points: NumberArray) {
-    const segmentLength = 3.4
-    const new_points = ResampleControlPoints(points, 3.4)
+export function getMatFromResamplePoints(points: NumberArray, segmentLength: number) {
+    //const segmentLength = 3.4
```

Comment:
please remove

Reply:
whats wrong in passing the segmentLength that depends on the curve type ?

Reply:
I meant removing the commented out code ` //const segmentLength = 3.4`